product_docs/docs/pem/8/installing_pem_agent/windows_agent.mdx

```diff
@@ -92,4 +92,4 @@ Where:
 -   `agent_name` specifies a descriptive name for the PEM agent.
 
 !!! Note
-    When configuring a shell/batch script run by a Windows agent that has PEM 7.11 or later version installed, set the `AllowBatchJobSteps` parameter to `True` in the `agent.cfg` file. The PEM agent doesn't execute any batch/shell script by default.
+    When configuring a shell/batch script run by a Windows agent that has PEM 8 or later version installed, set the `AllowBatchJobSteps` parameter to `True` in the `agent.cfg` file. The PEM agent doesn't execute any batch/shell script by default.
```

Comment:
@sonotley since we don't support 7.x anymore, I bumped this to 8 as well. 

Commenting here just to make sure this note is still relevant in 8 and 9 installations?

Reply:
I suggest we keep the note but remove the reference to the version, as it now applies to all supported versions.